utils/utils.ts

```diff
@@ -13,12 +13,29 @@ export function formatCurrency(amount: any, decimals = 2) {
       maximumFractionDigits: decimals,
     });
 
+    if (amount > 1_000_000_000) {
```

Comment:
```suggestion
    if (amount >= 1_000_000_000) {
```
Shall we check equal as well?